.github/workflows/test-pr-windows-macos.yml

```
@@ -0,0 +1,57 @@
+name: 🧪 Test (PR) (Windows, MacOS)
+
+# PRs touching create-remix/remix-dev will also run on Windows and OSX
+
+on:
+  pull_request:
+    paths-ignore:
+      - "packages/remix/**"
+      - "packages/remix-architect/**"
+      - "packages/remix-cloudflare/**"
+      - "packages/remix-cloudflare-pages/**"
+      - "packages/remix-cloudflare-workers/**"
+      - "packages/remix-css-bundle/**"
+      - "packages/remix-deno/**"
+      - "packages/remix-eslint-config/**"
+      - "packages/remix-express/**"
+      - "packages/remix-node/**"
+      - "packages/remix-react/**"
+      - "packages/remix-serve/**"
+      - "packages/remix-server-runtime/**"
+      - "packages/remix-testing/**"
+      - "docs/**"
+      - "scripts/**"
+      - "contributors.yml"
+      - "**/*.md"
```

This feels like we should just use `paths` instead?

```suggestion
    paths:
      - "packages/create-remix/**"
      - "packages/remix-dev/**"
    paths-ignore:
      - "**/*.md"
```